{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/graphql/users/mutations.py

```diff
@@ -0,0 +1,34 @@
+import graphene
+from django.db import transaction
+from graphene import relay
+from graphql import GraphQLError
+
+from .types import CurrentUser
+from {{cookiecutter.main_module}}.users import services as user_services
+
+
+class SignUp(relay.ClientIDMutation):
+
+    class Input:
+        email = graphene.String(required=True)
+        password = graphene.String(required=True)
+        first_name = graphene.String()
+        last_name = graphene.String()
+
+    @staticmethod
+    def validate_email(email):
+        if user_services.get_user_by_email(email):
+            raise GraphQLError('User with email already exists')
+
+    user = graphene.Field(CurrentUser)
+
+    @classmethod
+    @transaction.atomic
+    def mutate_and_get_payload(cls, root, info, **data):
+        cls.validate_email(data['email'])
+        user = user_services.create_user_account(**data)
+        return SignUp(user=user)
+
+
+class UserMutations(graphene.ObjectType):
+    signup = SignUp.Field()
```

Comment:
Let's move this main mutation to schema. This way, from each app, we import

```from myapp.schema import MyAppQueries, MyAppMutations```

Reply:
Done